publication-commons/src/main/java/no/unit/nva/publication/model/business/PublishingRequestCase.java

```diff
@@ -140,7 +140,6 @@ public void validateCreationRequirements(Publication publication)
         if (PublicationStatus.DRAFT_FOR_DELETION == publication.getStatus()) {
             throw new ConflictException(MARKED_FOR_DELETION_ERROR);
         }
-        assertThatPublicationHasMinimumMandatoryFields(publication);
```

Comment:
This line makes it impossible to update publication which contains only administrative agreements and misses entitydescription.reference.doi. We have an issue i TEST where user can not update publication because of it. 
We have to discuss if we want to have this check at all, evt. lack of what fields makes publication unpublishable

Reply:
If the reqs have changed, we should know this…